ocaml/xapi-storage/generator/lib/data.ml

```diff
@@ -239,14 +239,18 @@ end
 module Data (R : RPC) = struct
   open R
 
+  type copy_operation_v1 = string [@@deriving rpcty]
+
+  type mirror_operation_v1 = string [@@deriving rpcty]
+
   (** The primary key for referring to a long-running operation. *)
   type operation =
-    | Copy of uri * uri
-        (** Copy (src,dst) represents an on-going copy operation
-        from the [src] URI to the [dst] URI. *)
-    | Mirror of uri * uri
-        (** Mirror (src,dst) represents an on-going mirror
-        operation from the [src] URI to the [dst] URI. *)
+    | CopyV1 of copy_operation_v1
```

Comment:
why is there a v1?

Reply:
at @edwintorok 's recommendation to allow for future expansion